runtime/interpreter/value_storagecapabilitycontroller.go

```diff
@@ -209,6 +214,9 @@ func (v *StorageCapabilityControllerValue) ChildStorables() []atree.Storable {
 func (v *StorageCapabilityControllerValue) GetMember(inter *Interpreter, _ LocationRange, name string) Value {
 
 	switch name {
+	case sema.StorageCapabilityControllerTypeTagFieldName:
+		return v.GetTag()
```

Comment:
I guess caching the tag value is not possible here because there can be multiple copies of the same controller?

Reply:
Actually, there is only ever one controller instance, all `get` calls that load controller always return the same value. So yes, actually we can cache the value here in the value 👌 I'll add that

Reply:
Added in ebafc50c4397a4c929eea84e3a2b395379f4aa9c